redbot/cogs/trivia/trivia.py

```diff
@@ -625,7 +625,7 @@ async def _save_trivia_list(
         -------
         None
         """
-        filename = attachment.filename.rsplit(".", 1)[0]
+        filename = attachment.filename.rsplit(".", 1)[0].lower()
```

Comment:
Since Red supports i18n, it might be a better idea to use `.casefold()` here instead, which supports more characters than `.lower()` does.

Reply:
Ah makes sense thx.